docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+version: '3'
+
+services:
+
+  redis:
+    image: redis
+    container_name: redis
+    command: redis-server
+    ports:
+      - "6379:6379"
+
+  rabbitmq:
+      image: rabbitmq:3-management
+      container_name: rabbitmq
+      volumes:
+          - ./.docker/rabbitmq/logs/:/var/log/rabbitmq/
+      environment:
+          RABBITMQ_DEFAULT_USER: "sim"
+          RABBITMQ_DEFAULT_PASS: "simpass"
+          RABBITMQ_DEFAULT_VHOST: "/"
+      ports:
+          - 5672:5672
+          - 15672:15672
+
+  soda-sim:
+    image: soda-sim
+    volumes:
+      - /var/log/dolphin:/var/log/dolphin
+    ports:
+      - 8190:8190
+    environment:
+      - OS_COORDINATION__BACKEND_SERVER=192.168.20.158:6379
+      - OS_DEFAULT__TRANSPORT_URL=rabbit://sim:simpass@192.168.20.158:5672//
```

Comment:
Is it possible to get IP adrees from config or envirnment variable?

Reply:
Done